assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -744,8 +744,8 @@ module.exports = elementorModules.ViewModule.extend( {
 
 			if ( ! isSingleSlide ) {
 				swiperOptions.navigation = {
-					prevEl: $prevButton,
-					nextEl: $nextButton,
+					prevEl: $prevButton[ 0 ],
+					nextEl: $nextButton[ 0 ],
```

Comment:
Does this change not cause problems on the old swiper version?

Reply:
@hein-obox With manual testing, the old Swiper version is still working with this change. AFAIU the old version accepted a JQuery instance or an HTMLElement or CSSSelector.
The newest Libraries only accept HTMLElement or CSSSelector.
But I cannot find documentation on Swiper 5 to prove this.

Reply:
Would it be possible to distiguish between the old and the new version just for safety?

Reply:
@hein-obox It is not necessary :-)